pkg/virt-handler/non-root.go

```diff
@@ -0,0 +1,128 @@
+package virthandler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	diskutils "kubevirt.io/kubevirt/pkg/ephemeral-disk-utils"
+	hostdisk "kubevirt.io/kubevirt/pkg/host-disk"
+	"kubevirt.io/kubevirt/pkg/virt-handler/isolation"
+)
+
+func changeOwnershipOfHostDisks(vmiWithAllPVCs *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
```

Comment:
This is tricky. We either will need to require the right permissions/ownership and SELinux label or we leave this here.